src/components/formatters/test/CFormatterJsonBase_test.cc

```diff
@@ -48,54 +50,60 @@ using namespace NsSmartDeviceLink::NsJSONHandler::Formatters;
 TEST(CFormatterJsonBaseTest, JSonStringValueToSmartObj_ExpectSuccessful) {
   // Arrange value
   std::string string_val("test_string");
-  Json::Value json_value(string_val);  // Json value from string
+  utils::json::JsonValue json_value(string_val);
   SmartObject object;
   // Convert json to smart object
-  CFormatterJsonBase::jsonValueToObj(json_value, object);
+  utils::json::JsonValueRef json_value_ref = json_value;
```

Comment:
@OHerasym Whay do you need both `JsonValueRef ` and `JsonValue `? 
And if you still need both - may be use initialization instead of assignment?